extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -694,4 +719,100 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 }
 
+/**
+ * Keeps track of the third-party vendors' cross-domain iframes. Only one
+ * static instance will be created, which will keep manage ALL amp-analytics
+ * tags.
+ */
+class AmpAnalyticsFrames {
+  constructor() {
+    this.map = new Map();
+  }
+
+  /**
+   * Create a cross-domain iframe
+   * @param {!Element} parent  The document node of the parent page
+   * @param {!string} requestsFrameUrl  The URL, including data hash if
+   * applicable, of the cross-domain iframe
+   * @return {!Element}
+   */
+  create(parent, requestsFrameUrl) {
+    const context = {
+      "scriptSrc": "/examples/analytics-3p-remote-frame-helper.js",
```

Comment:
Agree. Also note that this is the flip side of my comment on examples/analytics-3p-remote-frame-helper.js line 17.